src/main/java/uk/gov/hmcts/cmc/claimstore/services/DirectionsQuestionnaireService.java

```diff
@@ -45,8 +46,9 @@ public DirectionsQuestionnaireService(PilotCourtService pilotCourtService) {
         if (isOptedForMediation(responseRejection)) {
             return Optional.of(REFERRED_TO_MEDIATION);
         }
+
         if (!isOnlineDQ(claim)) {
-            return Optional.empty();
+            return Optional.of(DIRECTIONS_QUESTIONNAIRE_DEADLINE);
```

Comment:
As we are no longer returning an empty option from this method I think we should change the signature to return a CaseEvent rather than an Optional<CaseEvent>

Reply:
Should I change that for all of them?